docs/c-runtime-library/reference/wctomb-s-wctomb-s-l.md

```diff
@@ -60,9 +60,11 @@ Error Conditions
 | `NULL` | >0 | `EINVAL` | not modified |
 | any | >`INT_MAX` | `EINVAL` | not modified |
 | any | too small | `EINVAL` | not modified |
-
+ 
 If any of the above error conditions occurs, the invalid parameter handler is invoked, as described in [Parameter validation](../parameter-validation.md). If execution is allowed to continue, `wctomb` returns `EINVAL` and sets `errno` to `EINVAL`.
 
+Return value `EILSEQ` indicates an encoding error has occurred. 
```

Comment:
Thank you for making this addition. Could we word it slightly differently and provide a link? For example: "An [`EILSEQ`](../errno-constants.md) return value indicates that the parameter `wchar` is not a valid wide character.

Reply:
The link may be hard to read since github interpreted the markdown. It was `[`EILSEQ`](../errno-constants.md)`

Reply:
with backticks around EILSEQ